packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -0,0 +1,86 @@
+import dynamic from 'next/dynamic';
+import React, { ReactElement, useContext, useState } from 'react';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import EmbedIcon from '../icons/Embed';
+import LinkIcon from '../icons/Link';
+import { ListIcon, SidebarMenuItem } from './common';
+import { Section, SectionCommonProps } from './Section';
+
+const SubmitArticleModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "SubmitArticleModal" */ '../modals/SubmitArticleModal'
+    ),
+);
+
+const NewSourceModal = dynamic(
+  () =>
+    import(/* webpackChunkName: "NewSourceModal" */ '../modals/NewSourceModal'),
+);
+
+export function ContributeSection(props: SectionCommonProps): ReactElement {
+  const {
+    canSubmitArticle,
+    submitArticleOn,
+    submitArticleSidebarButton,
+    submitArticleModalButton,
+  } = useContext(FeaturesContext);
+  const { trackEvent } = useContext(AnalyticsContext);
+  const [showSubmitArticle, setShowSubmitArticle] = useState(false);
+  const [showNewSourceModal, setShowNewSourceModal] = useState(false);
+  const contributeMenuItems: SidebarMenuItem[] = [];
```

Comment:
Let's actually set the suggest new source always as the default, we can then insert the submit article when it's existing at the right position.
No need to assign it later on.